lib/features/support/resilient_driver.rb

```diff
@@ -22,6 +24,9 @@ def initialize(username, access_key, local_id, target_device, app_location, loca
                                     app_location,
                                     locator,
                                     additional_capabilities
+
+    # This must appear after creation of @driver as AppAutomateDriver also does this
+    MazeRunner.driver = self
```

Comment:
Is it worth a unit test to check the right driver is assigned when `ResilientAppiumDriver`/`AppAutomateDriver` get created?

Reply:
Basic unit test now added - can also be enhanced over time.